app/experimenter/experiments/tests/test_forms.py

```diff
@@ -202,6 +205,302 @@ class Meta:
+            "type": Experiment.TYPE_ADDON,
+            "status": Experiment.STATUS_ACCEPTED,
+            "name": "This is an experiment!",
+            "short_description": "my short description",
+            "related_work": "some other experiment",
+            "proposed_duration": 15,
+            "proposed_enrollment": 5,
+            "addon_experiment_id": "add_on_experiment_id",
+            "addon_release_url": "https://example.com",
+            "pref_key": "a pref key",
+            "pref_type": "boolean",
+            "pref_branch": "default",
+            "public_name": "the public name",
+            "public_description": "the public description",
+            "population_percent": "50.0000",
+            "firefox_version": "55.0",
+            "firefox_channel": "Nightly",
+            "client_matching": "some random text about client matching",
+            "platform": "All Windows",
+            "objectives": "some blurb about an objective",
+            "analysis": "some blurb about analyis",
+            "survey_required": True,
+            "survey_urls": "https://example.com",
+            "survey_instructions": "some blurb about instructions",
+            "slug": "the-experiment-slug",
+            "normandy_id": 4321,
+            "data_science_bugzilla_url": "https://bugzilla.mozilla.org/123/",
+            "feature_bugzilla_url": "https://bugzilla.mozilla.org/124/",
+            "risk_internal_only": True,
+            "risk_partner_related": True,
+            "risk_brand": True,
+            "risk_fast_shipped": True,
+            "risk_confidential": True,
+            "risk_release_population": True,
+            "risk_revenue": True,
+            "risk_data_category": True,
+            "risk_external_team_impact": True,
+            "risk_telemetry_data": True,
+            "risk_ux": True,
+            "risk_security": True,
+            "risk_revision": True,
+            "risk_technical": True,
+            "risk_technical_description": "tech description",
+            "testing": "some blurb about testing",
+            "test_builds": "some blurb about test builds",
+            "qa_status": "some blurb qa status",
+            "review_science": True,
+            "review_engineering": True,
+            "review_qa_requested": True,
+            "review_intent_to_ship": True,
+            "review_bugzilla": True,
+            "review_qa": True,
+            "review_relman": True,
+            "review_advisory": True,
+            "review_legal": True,
+            "review_ux": True,
+            "review_security": True,
+            "review_vp": True,
+            "review_data_steward": True,
+            "review_comms": True,
+            "review_impacted_teams": True,
+        }
+
+        class TestForm(ChangeLogMixin, forms.ModelForm):
+
+            class Meta:
+                model = Experiment
+                fields = "__all__"
+
+        class TinyTestForm(ChangeLogMixin, forms.ModelForm):
```

Comment:
Tiny Teacup Dog approves of this Tiny Test Form

![Teacup-Dogs-These-will-melt-your-heart](https://user-images.githubusercontent.com/119884/59472223-969d3d00-8df2-11e9-8433-2fc17e1f2f87.jpg)